content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -98,6 +98,69 @@ appear. In this example, only :guilabel:`Marc Demo` meets these criteria.
    :align: center
    :alt: The employees with expert Javascript skills.
 
+Use case: Assess highest improvement
+====================================
+
+Another way to modify the :guilabel:`Appraisal Skills Report` is to identify the employee who has
+the highest amount of improved skills over a specific period of time.
+
+To view this information, first remove the default filter in the search bar. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Improvement` beneath the :icon:`fa-filter` :guilabel:`Filters` column. Enabling this
+filter only presents skills that have improved.
+
+It is possible to view the skills that have improved over a period of time, such as a specific
+quarter, or month. With the search bar drop-down menu still expanded, click :guilabel:`Add Custom
+Filter` at the bottom of the :icon:`fa-filter` :guilabel:`Filters` column, and an :guilabel:`Add
+Custom Filter` pop-up window appears.
+
+Select :guilabel:`Create Date` for the first drop-down field, then select :guilabel:`is between` for
+the second drop-down field. Once :guilabel:`is between` is selected, a second field appears after
+the last field. Using the calendar selector, select the date range to apply the filter to. Once all
+the fields are properly formatted, click :guilabel:`Add`.
+
+The custom filter presents only the skills that have improved during the specified time period,
+organized by employee.
+
+.. example::
+   To determine the employee with the most amount of improved skills for the third quarter, remove
+   the default filter in the search bar of the :guilabel:`Appraisal Skills Report`. Next, activate
+   the :guilabel:`Improvement` filter, then click :guilabel:`Add Custom Filter` at the bottom of the
+   :icon:`fa-filter` :guilabel:`Filters` column.
+
+   In the resulting :guilabel:`Add Custom Filter` pop-up window, select :guilabel:`Create Date` for
+   the first drop-down field, then select :guilabel:`is between` for the second drop-down field. Two
+   date fields appear after :guilabel:`is between` is selected.
+
+   Using the calendar selector, set the first date to :guilabel:`07/01/2024` and the second date to
+   :guilabel:`09/30/2024`, then click :guilabel:`Add`.
+
+   These filters present only the skills that have improved during the third quarter (between July
+   1st and September 30th, 2024), organized by employee.
+
+   .. image:: skills_evolution/custom-filter.png
+     :align: center
+     :alt: The Custom Filter pop-up with the parameters set.
```

Comment:
Please remove the :align: attribute from images per our [latest guidelines](https://www.odoo.com/documentation/18.0/contributing/documentation/rst_guidelines.html) :)
Also the :alt: needs one more space to be aligned :)
```suggestion
   .. image:: skills_evolution/custom-filter.png
      :alt: The Custom Filter pop-up with the parameters set.
```